src/plots/plot.ts

```diff
@@ -199,6 +204,16 @@ export class Plot extends Component {
   public setBounds(width: number, height: number, originX?: number, originY?: number) {
     super.setBounds(width, height, originX, originY);
     if (this._canvas != null) {
+      if (this._bufferCanvas && !this._bufferCanvasValid) {
```

Comment:
so, as soon as you change the size of a canvas with css or whatever, it clears the contents. we have some drag-resizable charts in various apps, and what was happening is that it would immediately do a full redraw while you drag-resize. This buffer canvas stores the current canvas contents when the resize starts and redraws it into the resized canvas. Eventually, the deferred rendering callback will trigger and do a full-rez redraw. If deferred rendering is disabled, the buffer copy will be overwritten immediately by a full redraw.